libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -810,7 +810,16 @@ bool AP_AHRS::_wind_estimate(Vector3f &wind) const
  */
 bool AP_AHRS::airspeed_sensor_enabled(void) const
 {
-    if (!AP_AHRS_Backend::airspeed_sensor_enabled()) {
+    return state.airspeed_estimate_type == AirspeedEstimateType::AIRSPEED_SENSOR;
```

Comment:
this looks circular reasoning? airspeed_estimate_type gets set based on airspeed_sensor_enabled()